http/src/main/java/com/redhat/lightblue/client/http/LightblueHttpClient.java

```diff
@@ -106,10 +112,13 @@ public LightblueHttpClient(String dataServiceURI, String metadataServiceURI, Boo
      * .client.request.LightblueRequest)
      */
     @Override
-    public LightblueResponse metadata(AbstractLightblueMetadataRequest lightblueRequest) {
+    public LightblueResponse metadata(LightblueRequest lightblueRequest) {
```

Comment:
I removed the `AbstractLightblue*Request` classes since they were no longer needed. Before they were used to encapsulate the HTTP method, but that's been moved to live with the rest of the request details in `LightblueRequest`. This means you can technically pass a request meant for a data node to `metadata`. If you don't like this we can add a marker interface to distinguish between metadata requests and data requests.